awx/ui_next/src/components/LaunchPrompt/LaunchPrompt.jsx

```diff
@@ -49,39 +44,89 @@ function LaunchPrompt({ config, resource, onLaunch, onCancel, i18n }) {
       : resource.extra_vars;
     setValue('extra_vars', mergeExtraVars(extraVars, surveyValues));
     setValue('scm_branch', values.scm_branch);
-    onLaunch(postValues);
+
+    onSubmit(postValues);
   };
+  const { error, dismissError } = useDismissableError(contentError);
+
+  if (error) {
+    return (
+      <AlertModal
+        isOpen={error}
+        variant="error"
+        title={i18n._(t`Error!`)}
+        onClose={() => {
+          dismissError();
+        }}
+      >
+        <ContentError error={error} />
+      </AlertModal>
+    );
+  }
+
+  return (
+    <Wizard
+      isOpen={!contentError}
```

Comment:
I think this will lead to some odd behavior if the user dismisses the error (the dismissable `error` will be cleared, but `contentError` will remain set, so nothing will render). I think you want `isOpen` to be true, since you've already returned an AlertModal if needed